pkg/server/testserver.go

```diff
@@ -691,6 +691,37 @@ func (ts *testServer) Activate(ctx context.Context) error {
 		return err
 	}
 
+	// If the test is indirectly dependent on rangefeed latency, but not
+	// particularly interested in observing the effects of the default
+	// rangefeed configuration, make the test faster.
+	if ts.params.FastRangefeeds {
+		ie := ts.InternalExecutor().(isql.Executor)
+		for _, cs := range []struct {
+			setting string
+			value   time.Duration
+		}{
+			{"kv.closed_timestamp.target_duration", 50 * time.Millisecond},
+			{"kv.closed_timestamp.side_transport_interval", 100 * time.Millisecond},
+			{"kv.rangefeed.closed_timestamp_refresh_interval", 100 * time.Millisecond},
+			{"kv.protectedts.poll_interval", 10 * time.Millisecond},
+			{"kv.protectedts.reconciliation.interval", 1 * time.Millisecond},
+			// Speed up span config reconciliation.
+			{"spanconfig.reconciliation_job.checkpoint_interval", 100 * time.Millisecond},
```

Comment:
These settings are unrelated to rangefeeds, they are for other mechanisms. We can include them here, but then the parameter should be called something else.